netlify.toml

```diff
@@ -1070,4 +1081,14 @@ status = 301
 [[redirects]]
 from = "/workflows/why-to-use-workflows"
 to = "/workflows"
+status = 301
+
+[[redirects]]
+from = "/integrations/appdome-integration"
+to = "/workflows/ios-specific-workflow-steps/appdome-build-to-secure-for-ios"
+status = 301
```

Comment:
It seems that it has one reference from "release notes" and it's about **Android**.

![Screenshot 2024-12-02 at 14 20 05](https://github.com/user-attachments/assets/bc4bb57a-0ff0-4088-b21a-7e9ebd0b0fec)


Instead of adding redirect, it will be better to fix the URL itself in release notes.